app/models/concerns/orchestration/common.rb

```diff
@@ -5,4 +5,9 @@ def handle_validation_errors
     logger.debug "Error occured during validations of #{self.class.name}: #{e.message}"
     nil
   end
+
+  # override sql logger for all orchestration mix-ins
+  def logger
```

Comment:
Katello uses a common class for all models of the plugin. https://github.com/Katello/katello/blob/master/app/models/katello/model.rb. Maybe time for Foreman to follow the same approach to influence the common logic. I assume you don't want to iterate thought the models dynamically, but rather putting the `include` statement there, right?